lib/utils/matrix_sdk_extensions/matrix_file_extension.dart

```diff
@@ -61,6 +38,42 @@ extension MatrixFileExtension on MatrixFile {
     return;
   }
 
+  void downloadImageInWeb(BuildContext context) {
```

Comment:
if it is not mobile, you can use this library instead
https://pub.dev/packages/file_saver

Reply:
it worked for web, also desktop in the future

Reply:
IMO: No need

Reply:
great lib, it's better than native download. had to do some extensions to support mimetype tho

Reply:
9519e309ee9780034b3d82838f02d9d815f2a81b